pkg/asset/machines/master.go

```diff
@@ -91,7 +91,7 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 		mpool.Set(ic.Platform.Libvirt.DefaultMachinePlatform)
 		mpool.Set(pool.Platform.Libvirt)
 		pool.Platform.Libvirt = &mpool
-		m.MachinesDeprecated, err = libvirt.Machines(clusterID.ClusterID, ic, &pool, "master", "master-user-data")
+		m.Machines, err = libvirt.Machines(clusterID.ClusterID, ic, &pool, "master", "master-user-data")
```

Comment:
This needs an associated change [here][1].

[1]: https://github.com/openshift/installer/blob/193a3f2cb7b8da375da90126ac4757254cdb17fe/pkg/asset/cluster/tfvars.go#L120